modules/apps/portal-language/portal-language-lang/src/main/resources/content/Language.properties

```diff
@@ -20114,6 +20117,7 @@ token=Token
 token-configuration-name=Token Based SSO
 token-connection-timeout=Token Connection Timeout in Milliseconds
 token-connection-timeout-help=Set the amount of time to wait (in milliseconds) when establishing a connection to validate the token before timing out. A value of 0 means infinity and is not recommended.
+token-definition-type=Token definition type
```

Comment:
@thiago-buarqque 
For all the language keys, I just copied them directly from the Figma here:
https://www.figma.com/design/4oeQva5ooVkUyAaAgfBQmY/LPD-30360?node-id=178-8762&node-type=canvas&t=WT6VWDL5XVLg77ih-0
Should I still change them for your suggestions?

Reply:
I see.

Hmm I believe 

```
based-on-token-definition=Based on Token Definition
```

should be 

```
based-on-x-frontend-token-definition=Based on {1} frontend token definition
```

This is regarding the description in the style book card, right?

![image](https://github.com/user-attachments/assets/a8d294a7-971f-4212-99af-56ff0200d6a2)

Since it's not a title/label it should be lowercase. I know the design says that, but Brian will change it when he seems it.

Reply:
@SelenaAungst also, I don't see where `Token definition type` is used

Reply:
"Based on Token Definition" is actually the [Label for one of the inputs in the creation modal](https://www.figma.com/design/4oeQva5ooVkUyAaAgfBQmY/LPD-30360?node-id=178-8763&node-type=frame&t=oRm0l6HwmR0MCNwi-0). So its similar but a different language key then the one you're thinking of @thiago-buarqque . Thanks though.

Reply:
I think I removed where I used token-definition-type, I'll get rid of the key for that @thiago-buarqque 

As for the other key, it looks like @marcoscv-work and @ethib137 discussed this on the figma design and decided on "Create Style Book for" instead of "Based on Token Definition", so I will change it to that instead

Reply:
Awesome, thanks @SelenaAungst